HelloWars/Arena/Pages/Converters/BoolToOpacityConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Windows.Data;
+
+namespace Arena.Pages.Converters
+{
+    public sealed class BoolToOpacityConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            return (value is bool && (bool)value) ? 1 : 0.4;
```

Comment:
0.4 looks nice. :) after converting it to Opacity in CompetitorViewControl. (Used while user is 'dead')
Exception isn't necessary, this is only a simple converter.